docs/io-cloud-storage-sink.md

```diff
@@ -1,58 +1,52 @@
 ---
-description: The Cloud Storage sink connector pulls messages from Pulsar topics and persists messages to Cloud Storage.
-author: ["ASF"]
-contributors: ["ASF"]
-language: Java
-document: 
-source: "https://github.com/streamnative/pulsar-io-cloud-storage/tree/v2.5.1/src/main/java/org/apache/pulsar/io/jcloud"
-license: Apache License 2.0
-tags: ["Pulsar IO", "Cloud Storage", "Sink"]
-alias: Cloud Storage Sink
-features: ["Use the Cloud Storage sink connector to sync data from Pulsar"]
-license_link: "https://www.apache.org/licenses/LICENSE-2.0"
-icon: "https://a0.awsstatic.com/libra-css/images/logos/aws_logo_smile_179x109.png"
-download: "https://github.com/streamnative/pulsar-io-cloud-storage/releases/download/v2.5.1/pulsar-io-cloud-storage-2.5.1.nar"
-support: StreamNative
-support_link: https://streamnative.io
-support_img: "/images/connectors/streamnative.png"
-dockerfile: 
-owner_name: ""
-owner_img: ""
-id: "io-cloud-storage-sink"
+dockerfile: "https://hub.docker.com/r/streamnative/pulsar-io-cloud-storage"
+alias: Cloud Storage Sink Connector
 ---
 
-Cloud Storage Sink connector for Pulsar
+The Cloud Storage sink connector supports exporting data from Pulsar topics to cloud storage (such as AWS S3 and Google GCS) either in Avro, JSON, Parquet or other formats. According to your environment, the Cloud Storage sink connector can guarantee exactly-once support for exporting data to cloud storage.
 
-The Cloud Storage Sink connector supports exporting data from Pulsar Topic to cloud storage in Avro, json, parquet and other formats, such as aws-s3, google-GCS, etc. According to your environment, the Cloud Storage sink connector can guarantee exactly-once support for exporting to cloud storage.
+![](/images/connectors/cloud-storage-sink.png)
```

Comment:
This image does not exist yet

Reply:
I can help to draw the image, and the image link should be changed to "/docs/cloud-storage-sink.png".

Reply:
Thanks for helping out with the image @Huanli-Meng. I will change the link and wait to add the image then